httpserver-rs/Cargo.toml

```diff
@@ -1,5 +1,8 @@
 [package]
-name = "wasmcloud-provider-httpserver"
+name = "wasmcloud-httpserver-provider"
```

Comment:
Every other provider uses `wasmcloud-provider-X`, including the [http server action](https://github.com/wasmCloud/capability-providers/blob/c68fe646861459cdcb30db42c908f4e4716756cc/.github/workflows/httpserver.yml#L156). Why not just update the lib action to match?